app/assets/stylesheets/govuk_publishing_components/components/_textarea.scss

```diff
@@ -1 +1,5 @@
 @import "govuk-frontend/components/textarea/textarea";
+
+.gem-c-textarea .govuk-textarea {
+  margin-bottom: 0;
```

Comment:
In the default component state you're correct, we don't need it, because both the `.govuk-textarea` element and the wrapping div have the same bottom margin. However, if you pass a `margin_bottom` value that results in a smaller margin bottom for the overall component (e.g. zero), the `.govuk-textarea` would still have that margin bottom, effectively overriding the margin zero on the wrapping div.

Reply:
Oh.. because of the `govuk-form-group` having its own spacing defined.. argh.. that's ugly. Let's get it merged and we'll think of a way to approach form elements with this wrapper.